packages/web/src/common/store/cache/collections/utils/retrieveCollections.ts

```diff
@@ -248,24 +245,32 @@ export function* retrieveCollections(
     },
     getEntriesTimestamp: selectEntriesTimestamp,
     retrieveFromSource: function* (ids: ID[]) {
-      const audiusBackendInstance = yield* getContext('audiusBackendInstance')
+      const audiusSdk = yield* getContext('audiusSdk')
+      const sdk = yield* call(audiusSdk)
       let metadatas: CollectionMetadata[]
 
       if (ids.length === 1) {
-        metadatas = yield* call(retrieveCollection, { playlistId: ids[0] })
+        const { data = [] } = yield* call(
+          [sdk.full.playlists, sdk.full.playlists.getPlaylist],
+          {
+            playlistId: Id.parse(ids[0]),
+            userId: OptionalId.parse(userId)
+          }
+        )
+        const [collection] = transformAndCleanList(
+          data,
+          userCollectionMetadataFromSDK
+        )
+        metadatas = [collection]
       } else {
-        // TODO: Remove this branch when we have batched endpoints in new V1 api.
```

Comment:
I think I wrote this TODO. Technically I think this is still a potential issue. It just has a higher limit since the query params will go from `playlist_id=` to `id=`. But if they are still repeated for each entry, it could cause a problem for bulk fetching, say, 200 playlists?